griptape/drivers/prompt/google_prompt_driver.py

```diff
@@ -110,19 +110,18 @@ def try_stream(self, prompt_stack: PromptStack) -> Iterator[DeltaMessage]:
                 )
 
     def _base_params(self, prompt_stack: PromptStack) -> dict:
-        GenerationConfig = import_optional_dependency("google.generativeai.types").GenerationConfig
-        ContentDict = import_optional_dependency("google.generativeai.types").ContentDict
-        Part = import_optional_dependency("google.generativeai.protos").Part
+        types = import_optional_dependency("google.generativeai.types")
+        protos = import_optional_dependency("google.generativeai.protos")
```

Comment:
I liked it better before, but 🤷

Reply:
Yeah this one is a little weird, but I think I'd rather go for consistency.